app/sidekiq/bgs/submit_form674_job.rb

```diff
@@ -9,45 +9,52 @@ class Invalid674Claim < StandardError; end
+      end
+    end
 
-    # rubocop:disable Metrics/MethodLength
-    # method length is currently disabled due to the two flippers currently enabled in this method.
-    # when both are resolved we will be able to remove the method length disabling.
-    # BlockNesting is also disabled for the same reason.
     def perform(user_uuid, icn, saved_claim_id, vet_info, user_struct_hash = {})
       Rails.logger.info('BGS::SubmitForm674Job running!', { user_uuid:, saved_claim_id:, icn: })
+
+      @vet_info = vet_info
+      @user = generate_user_struct
+      @user_uuid = user_uuid
+      @saved_claim_id = saved_claim_id
+
       in_progress_form = InProgressForm.find_by(form_id: FORM_ID, user_uuid:)
-      in_progress_copy = in_progress_form_copy(in_progress_form)
-      claim_data = valid_claim_data(saved_claim_id, vet_info)
-      normalize_names_and_addresses!(claim_data)
-      user_struct = user_struct_hash.present? ? OpenStruct.new(user_struct_hash) : generate_user_struct(vet_info['veteran_information']) # rubocop:disable Layout/LineLength
+      @in_progress_copy = in_progress_form_copy(in_progress_form)
+
+      claim_data = normalize_names_and_addresses!(valid_claim_data)
+
+      @user = user_struct_hash.present? ? OpenStruct.new(user_struct_hash) : generate_user_struct
 
-      user = user_struct
-      BGS::Form674.new(user).submit(claim_data)
+      BGS::Form674.new(user, claim).submit(claim_data)
 
-      send_confirmation_email(user)
+      send_confirmation_email
       in_progress_form&.destroy
       Rails.logger.info('BGS::SubmitForm674Job succeeded!', { user_uuid:, saved_claim_id:, icn: })
     rescue => e
       Rails.logger.error('BGS::SubmitForm674Job failed!', { user_uuid:, saved_claim_id:, icn:, error: e.message })
       log_message_to_sentry(e, :error, {}, { team: 'vfs-ebenefits' })
-      salvage_save_in_progress_form(FORM_ID, user_uuid, in_progress_copy)
-      if Flipper.enabled?(:dependents_central_submission)
-        user_struct = user_struct_hash.present? ? OpenStruct.new(user_struct_hash) : generate_user_struct(vet_info['veteran_information']) # rubocop:disable Layout/LineLength
-        CentralMail::SubmitCentralForm686cJob.perform_async(saved_claim_id,
-                                                            KmsEncrypted::Box.new.encrypt(vet_info.to_json),
-                                                            KmsEncrypted::Box.new.encrypt(user_struct.to_h.to_json))
-      else
-        DependentsApplicationFailureMailer.build(user).deliver_now if user&.email.present? # rubocop:disable Style/IfInsideElse
-      end
+
+      raise
     end
-    # rubocop:enable Metrics/MethodLength
 
     private
 
-    def valid_claim_data(saved_claim_id, vet_info)
-      claim = SavedClaim::DependencyClaim.find(saved_claim_id)
+    def valid_claim_data
+      @claim = SavedClaim::DependencyClaim.find(saved_claim_id)
```

Comment:
with claim in the attr_reader, is getting @claim still necessary?

Reply:
creating an instance variable is necessary for this line
https://github.com/department-of-veterans-affairs/vets-api/blob/827b92032ed9f5d093ac9cc9fe3b87b8f445ef13/app/sidekiq/bgs/submit_form674_encrypted_job.rb#L45